wgpu-hal/src/vulkan/instance.rs

```diff
@@ -609,6 +609,11 @@ impl crate::Instance<super::Api> for super::Instance {
         let obs_layer = CStr::from_bytes_with_nul(b"VK_LAYER_OBS_HOOK\0").unwrap();
         let has_obs_layer = find_layer(&instance_layers, obs_layer).is_some();
 
+        let portability_enumeration_layer =
+            CStr::from_bytes_with_nul(b"VK_KHR_portability_enumeration\0").unwrap();
+        let has_portability_enumeration_layer =
```

Comment:
We are looking for layer that is also an extension? Could you add some doc comments explaining why we're doing this/vk is set up like this

Reply:
D'oh - thanks for paying attention.